### PR TITLE
Docs to install traffic managers in multiple namespaces

### PR DESCRIPTION
**Pull request by josecv**

### CODE REVIEW

File: docs/pre-release/reference/config.md [hunk 1]
```diff
@@ -110,3 +111,22 @@ clusters:
...
+clusters:
+- cluster:
+    server: https://127.0.0.1
+    extensions:
+    - name: telepresence.io
+      extension:
+        manager:
+          namespace: staging
+  name: example-cluster
+```
```
Comment (josecv) [inline docs/pre-release/reference/config.md:132]: in addition to this should we add a `--namespace` argument to `telepresence connect`? that way users can switch back and forth without having to come back to this file

Comment (donnyyung) [reply docs/pre-release/reference/config.md:132]: I think that would be a good idea but not necessary for v0... so all to say, I think you can leave it out of the docs for now and then we can add it if we decide to add that feature.  

In general, I think it'd be nice if _all_ the config values could be set with the command line, I think some folks have requested that for timeouts and such.  (not saying you need to take that on, but it is definitely something we have wanted to do :) )

File: docs/pre-release/install/helm.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ For more details on what the Helm chart installs and what can be configured, tak
 The Helm chart supports being installed into any namespace, not necessarily `ambassador`. Simply pass a different `namespace` argument to `helm install`:
```
Comment (donnyyung) [inline docs/pre-release/install/helm.md:35]: I would add a line spelling it out here: "For example, if you wanted to deploy the traffic manager to the `staging` namespace:"

File: docs/pre-release/reference/config.md (hunk 1 above)
Comment (josecv) [reply docs/pre-release/reference/config.md:132]: ok, i think it makes sense to keep it out for now then :) especially if we might later want to come up with a way to do it for all configs

File: .gitignore [hunk 3]
```diff
@@ -11,3 +11,4 @@
 
 # Other
 *.log
+.*.swp
```
Comment (LukeShu) [inline .gitignore:14]: Tip: Put this type of thing in `~/.config/git/ignore` so you don't have to add it to every project you contribute to.

File: docs/pre-release/install/helm.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+## Before you begin
+
+The Telepresence Helm chart is hosted by Ambassador Labs and published at `https://www.getambassador.io`.
+
+Start by adding this repo to your Helm client with the following command:
+
+```
+helm repo add datawire https://www.getambassador.io
+```
```
Comment (LukeShu) [inline docs/pre-release/install/helm.md:13]: Probably mark this (and the other snippets) as <code>\`\`\`shell</code> ?

File: docs/pre-release/releaseNotes.yml [hunk 5]
```diff
@@ -33,6 +33,34 @@ items:
...
+      # Headliners
+      - type: feature
+        title: Traffic-manager Helm Chart
+        body: >-
+          We now support installing the traffic-manager via helm.
+          This will make it easier for operators to see what the
+          traffic-manager needs to install so that telepresence works
+          correctly, as well as providing a more direct route to customizing
+          installation.
+        image: ./telepresence-2.3.3-helm.png
```
Comment (LukeShu) [inline docs/pre-release/releaseNotes.yml:45]: Am I right that that image exists on a branch in getambassador.io.git?  Please copy it to telepresence.io.git `docs/pre-release/release-notes/`

File: .gitignore (hunk 3 above)
Comment (josecv) [reply .gitignore:14]: aah nice, didn't know about that

File: docs/pre-release/releaseNotes.yml (hunk 5 above)
Comment (josecv) [reply docs/pre-release/releaseNotes.yml:45]: ah yes, good point, forgot to bring it over when i copied Donny's changes into this branch

**Review by LukeShu — APPROVED** [review]

**Comment by LukeShu** [comment]
Rebase on to `lukeshu/pull-docs` to get CI to pass.

**Review by donnyyung — APPROVED** [review]

**Comment by donnyyung** [comment]
My force push was just rebasing this since jose is OOO today and tomorrow. Merging this in now!

**Pull request merged**